apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -13,15 +13,17 @@ const Header = (): JSX.Element => {
 		return <></>;
 	}
 
-	const userName = authenticatedUser.name;
+	const { email, name } = authenticatedUser;
 
 	return (
 		<header className={styles["header"]}>
 			<div className={styles["logo-container"]}>
 				<img alt="GitFit logo" className={styles["logo-img"]} src={logoSrc} />
 				<span className={styles["logo-text"]}>Logo</span>
 			</div>
-			<Avatar name={userName} />
+			<Popover content={<UserPopover email={email} name={name} />}>
```

Comment:
I am not sure if we need a separate component for the `UserPopover` or at least we do not need to put it in the `libs` since this is not a common component

let's remove this component or put it near the header component at least

Reply:
Agree, I think I'll put it near the header, so we have styles and tsx of UserPopover in a separated from header files to not litter the header's code.

Reply:
Done